README.md

```diff
@@ -9,9 +9,9 @@ Eleventy plugin which provides common filters, shortcodes and transforms for [Fl
 
 ## Requirements
 
-- Node >= 18
-- Eleventy >= 2.0.1
-- Infusion >= 4.6.0
+- Node >= 20
```

Comment:
Since this is a breaking change release I don't think it's unreasonable to bump the Node requirement. Upgrading Node on our Eleventy projects has not typically caused any issues.